src/applications/vaos/services/location/transformers.js

```diff
@@ -265,3 +265,14 @@ export function setSupportedSchedulingMethods({
 export function transformFacilities(facilities) {
   return facilities.map(transformFacility);
 }
+
+/**
+ * Returns formatted address from facility details object
+ *
+ * @param {*} facility - facility details object
+ */
+export function formatFacilityAddress(facility) {
```

Comment:
I'd put this in location/index.js rather than here, since it's formatting a FHIR location into a human readable string, rather than transforming data into the FHIR format.